src/nvim/testdir/setup.vim

```diff
@@ -0,0 +1,23 @@
+" Common preparations for running tests.
+
+" This also enables use of line continuation.
+set viminfo+=nviminfo
+
+" Make sure 'runtimepath' does not include $HOME.
+set rtp=$VIM/vimfiles,$VIMRUNTIME,$VIM/vimfiles/after
+
+" Make sure $HOME does not get read or written.
+let $HOME = '/does/not/exist'
+
+" Avoid stopping at the "hit enter" prompt
+set nomore
+
+" Output all messages in English.
+language mess C
+
+" Always use forward slashes.
+set shellslash
+
+" Align with vim defaults.
+set directory^=.
+set nohidden
```

Comment:
Why was all this other stuff put into setup.vim? Upstream Vim doesn't have this, and the patch doesn't have this. The only purpose for us having setup.vim at all, is to make merges easier--so making it different doesn't achieve it.